src/app/layout.tsx

```diff
@@ -27,7 +16,10 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body
-        className={`${geistSans.variable} ${geistMono.variable} antialiased`}
+        className="antialiased"
+        style={{
```

Comment:
When we use next/fonts/google lib we can use it the way it was with local fonts. I'll fix later